<?xml version="1.0" encoding="UTF-8"?>
<robot generator="Robot 6.1.1 (Python 3.11.5 on win32)" generated="20231114 12:09:59.480" rpa="false" schemaversion="4">
<suite id="s1" name="Prp-Test-01" source="C:\Users\BKU.SE\Desktop\SpectralEngines\Testing\Testautomation\testautomation-e2e\Tests\prp-test-01.robot">
<test id="s1-t1" name="Open_Application" line="10">
<kw name="Open Application" library="AppiumLibrary">
<arg>http://127.0.0.1:4724</arg>
<arg>platformName=Android</arg>
<arg>deviceName=RF8T90V5A7H</arg>
<arg>appPackage=com.purplepro.app</arg>
<arg>appActivity=com.purplepro.app.MainActivity</arg>
<arg>automationName=Uiautomator2</arg>
<doc>Opens a new application to given Appium server.
Capabilities of appium server, Android and iOS,
Please check https://github.com/appium/appium/blob/master/docs/en/writing-running-appium/server-args.md
| *Option*            | *Man.* | *Description*     |
| remote_url          | Yes    | Appium server url |
| alias               | no     | alias             |
| strict_ssl          | No     | allows you to send commands to an invalid certificate host like a self-signed one. |</doc>
<msg timestamp="20231114 12:10:01.649" level="WARN">Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('&lt;urllib3.connection.HTTPConnection object at 0x000001ED92646050&gt;: Failed to establish a new connection: [WinError 10061] Es konnte keine Verbindung hergestellt werden, da der Zielcomputer die Verbindung verweigerte')': /session</msg>
<msg timestamp="20231114 12:10:03.681" level="WARN">Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('&lt;urllib3.connection.HTTPConnection object at 0x000001ED8EC5E910&gt;: Failed to establish a new connection: [WinError 10061] Es konnte keine Verbindung hergestellt werden, da der Zielcomputer die Verbindung verweigerte')': /session</msg>
<msg timestamp="20231114 12:10:05.715" level="WARN">Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('&lt;urllib3.connection.HTTPConnection object at 0x000001ED8EC5EA10&gt;: Failed to establish a new connection: [WinError 10061] Es konnte keine Verbindung hergestellt werden, da der Zielcomputer die Verbindung verweigerte')': /session</msg>
<kw name="Capture Page Screenshot" library="AppiumLibrary">
<doc>Takes a screenshot of the current page and embeds it into the log.</doc>
<msg timestamp="20231114 12:10:07.751" level="FAIL">No application is open</msg>
<status status="FAIL" starttime="20231114 12:10:07.750" endtime="20231114 12:10:07.763"/>
</kw>
<msg timestamp="20231114 12:10:07.764" level="WARN">Keyword 'Capture Page Screenshot' could not be run on failure: No application is open</msg>
<msg timestamp="20231114 12:10:07.765" level="FAIL">MaxRetryError: HTTPConnectionPool(host='127.0.0.1', port=4724): Max retries exceeded with url: /session (Caused by NewConnectionError('&lt;urllib3.connection.HTTPConnection object at 0x000001ED92652690&gt;: Failed to establish a new connection: [WinError 10061] Es konnte keine Verbindung hergestellt werden, da der Zielcomputer die Verbindung verweigerte'))</msg>
<status status="FAIL" starttime="20231114 12:09:59.597" endtime="20231114 12:10:07.767"/>
</kw>
<kw name="Wait Until Page Contains Element" library="AppiumLibrary">
<arg>${SIGNIN-BUTTON}</arg>
<doc>Waits until element specified with `locator` appears on current page.</doc>
<status status="NOT RUN" starttime="20231114 12:10:07.767" endtime="20231114 12:10:07.767"/>
</kw>
<kw name="Click Element" library="AppiumLibrary">
<arg>${SIGNIN-BUTTON}</arg>
<doc>Click element identified by `locator`.</doc>
<status status="NOT RUN" starttime="20231114 12:10:07.767" endtime="20231114 12:10:07.767"/>
</kw>
<kw name="Sleep" library="BuiltIn">
<arg>2</arg>
<doc>Pauses the test executed for the given time.</doc>
<status status="NOT RUN" starttime="20231114 12:10:07.767" endtime="20231114 12:10:07.767"/>
</kw>
<kw name="Wait Until Page Contains" library="AppiumLibrary">
<arg>text=Konto ausw√§hlen</arg>
<doc>Waits until `text` appears on current page.</doc>
<status status="NOT RUN" starttime="20231114 12:10:07.767" endtime="20231114 12:10:07.767"/>
</kw>
<kw name="Sleep" library="BuiltIn">
<arg>2</arg>
<doc>Pauses the test executed for the given time.</doc>
<status status="NOT RUN" starttime="20231114 12:10:07.767" endtime="20231114 12:10:07.767"/>
</kw>
<kw name="Click Element" library="AppiumLibrary">
<arg>${ACCOUNT}</arg>
<doc>Click element identified by `locator`.</doc>
<status status="NOT RUN" starttime="20231114 12:10:07.767" endtime="20231114 12:10:07.767"/>
</kw>
<kw name="Wait Until Page Contains Element" library="AppiumLibrary">
<arg>//android.widget.TextView[@text="Kennwort eingeben"]</arg>
<doc>Waits until element specified with `locator` appears on current page.</doc>
<status status="NOT RUN" starttime="20231114 12:10:07.768" endtime="20231114 12:10:07.768"/>
</kw>
<kw name="Click Element" library="AppiumLibrary">
<arg>${KENNWORT}</arg>
<doc>Click element identified by `locator`.</doc>
<status status="NOT RUN" starttime="20231114 12:10:07.768" endtime="20231114 12:10:07.768"/>
</kw>
<kw name="Input Text" library="AppiumLibrary">
<arg>${KENNWORT}</arg>
<arg>text=123456</arg>
<doc>Types the given `text` into text field identified by `locator`.</doc>
<status status="NOT RUN" starttime="20231114 12:10:07.768" endtime="20231114 12:10:07.768"/>
</kw>
<status status="FAIL" starttime="20231114 12:09:59.597" endtime="20231114 12:10:07.768">MaxRetryError: HTTPConnectionPool(host='127.0.0.1', port=4724): Max retries exceeded with url: /session (Caused by NewConnectionError('&lt;urllib3.connection.HTTPConnection object at 0x000001ED92652690&gt;: Failed to establish a new connection: [WinError 10061] Es konnte keine Verbindung hergestellt werden, da der Zielcomputer die Verbindung verweigerte'))</status>
</test>
<status status="FAIL" starttime="20231114 12:09:59.480" endtime="20231114 12:10:07.769"/>
</suite>
<statistics>
<total>
<stat pass="0" fail="1" skip="0">All Tests</stat>
</total>
<tag>
</tag>
<suite>
<stat pass="0" fail="1" skip="0" id="s1" name="Prp-Test-01">Prp-Test-01</stat>
</suite>
</statistics>
<errors>
<msg timestamp="20231114 12:10:01.649" level="WARN">Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('&lt;urllib3.connection.HTTPConnection object at 0x000001ED92646050&gt;: Failed to establish a new connection: [WinError 10061] Es konnte keine Verbindung hergestellt werden, da der Zielcomputer die Verbindung verweigerte')': /session</msg>
<msg timestamp="20231114 12:10:03.681" level="WARN">Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('&lt;urllib3.connection.HTTPConnection object at 0x000001ED8EC5E910&gt;: Failed to establish a new connection: [WinError 10061] Es konnte keine Verbindung hergestellt werden, da der Zielcomputer die Verbindung verweigerte')': /session</msg>
<msg timestamp="20231114 12:10:05.715" level="WARN">Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('&lt;urllib3.connection.HTTPConnection object at 0x000001ED8EC5EA10&gt;: Failed to establish a new connection: [WinError 10061] Es konnte keine Verbindung hergestellt werden, da der Zielcomputer die Verbindung verweigerte')': /session</msg>
<msg timestamp="20231114 12:10:07.764" level="WARN">Keyword 'Capture Page Screenshot' could not be run on failure: No application is open</msg>
</errors>
</robot>
